### DISCUSSION TITLE
Identifying genes regulated by a TF

### DISCUSSION THREAD

**Discussion by galib36** [GreenleafLab/ArchR#863] [Questions / Documentation]
Hello,
Is there any way to identify the genes that are regulated by a certain TF. Something similar to what `correlateMatrices()` does, but on all the genes?

Thank you.

**Comment by rcorces** [comment]
You would have to code that yourself. the motif annotations tell you which motifs are present in which peaks. you could cross reference this with co-accessibility or peak to gene links.
